workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -19,9 +19,7 @@ public class Bootstrap : MonoBehaviour
 
         public const string LoggerName = "Bootstrap";
 
-        private static readonly List<WorkerBase> Workers = new List<WorkerBase>();
-
-        private static ConnectionConfig connectionConfig;
+        private static Dictionary<SpatialOSWorld, ConnectionConfig> WorldToConfig = new Dictionary<SpatialOSWorld, ConnectionConfig>();
```

Comment:
It feels weird to store the connection config here. 

Is there a reason the world/worker doesn't own it?

Reply:
not really, still trying to figure out whether the world or the worker should own it. probably world

in any case, I'll change it. this was more to get the game running and check if the new worker abstraction actually works